modules/issue/template/template.go

```diff
@@ -385,6 +424,22 @@ func (o *valuedOption) IsChecked() bool {
 	return false
 }
 
+func (o *valuedOption) VisibleInContent() bool {
+	if o.field.Type == api.IssueFormFieldTypeCheckboxes {
```

Comment:
Erm… is this check still necessary?

Reply:
yes ... and as we don't have only an `interface{}` type we have to do it that way :/

we might in the future declare types and de-/serialize it based on them directly

-> I'll create an issue to track this. should not be that hard to do ;)

Reply:
done: #29569